.github/workflows/unit_tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
Did you see https://github.com/mne-tools/mne-bids/issues/933#issuecomment-997968319 ?

Reply:
Nope. Argh. Ok will adjust! Thanks.